packages/flutter_tools/lib/src/ios/mac.dart

```diff
@@ -165,7 +165,6 @@ Future<XcodeBuildResult> buildXcodeProject({
       logger: globals.logger,
       fileSystem: globals.fs,
       plistParser: globals.plistParser,
-      features: featureFlags,
```

Comment:
FYI, this was added so that we could split the SPM app migration into its own feature. This would let us turn on SPM by default for new apps/plugins while keeping the migration opt-in until we had higher confidence in it.

For details, see: https://github.com/flutter/flutter/issues/151567#issuecomment-2455941279